src/components/Header/Header.tsx

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+
+import { TodoUpdateContext, TodosContext } from '../../context/TodosContext';
+import { Todo } from '../../types/Todo';
+import { USER_ID } from '../../constants/USER_ID';
```

Comment:
it looks like you create file only for one constant and called it `USER_ID.ts`

this file sholud have a name like `commonConsts.ts` for example

Reply:
in that case I thought that I have only one const, so named like that. It's not correct?